client/scripts/index.js

```diff
@@ -1,3 +1,5 @@
+let pad = null
```

Comment:
you can put a TODO here 
you made it global to ignore scoping
but eventually you can pass around the pad to each function that needs it to control the scope

Reply:
and add a trello ticket to refactor `pad`'s scoping